nativelib/src/main/scala/scala/scalanative/native/time.scala

```diff
@@ -0,0 +1,27 @@
+package scala.scalanative
+package native
+
+@extern
+object time {
+
+  type time_t = CLong
+
+  @struct
+  class timespec(
+      val tv_sec: time_t = 0L, /* seconds */
+      val tv_nsec: CLong = 0L /* nanoseconds */
+  )
+
+  def clock_getres(clk_id: Clock.clockid_t, res: Ptr[timespec]): CInt  = extern
+  def clock_gettime(clk_id: Clock.clockid_t, res: Ptr[timespec]): CInt = extern
+  def clock_settime(clk_id: Clock.clockid_t, res: Ptr[timespec]): CInt = extern
```

Comment:
It looks like those are [not C11 libc functions](http://www.cplusplus.com/reference/ctime/), but posix time functions. `@extern object time` should only contain the bindings to things defined in `time.h`. We also can't unconditionally depend on posix in our core libraries as that would complicate Windows support.